src/Asv.Common/Units/GeoPoint/GeoPoint.cs

```diff
@@ -105,9 +102,18 @@ public override string ToString()
 
         public bool Equals(GeoPoint other)
         {
-            return Longitude.Equals(other.Longitude) && Latitude.Equals(other.Latitude) && Altitude.Equals(other.Altitude);
+            return Longitude.ApproximatelyEquals(other.Longitude) 
+                   && Latitude.ApproximatelyEquals(other.Latitude) 
+                   && Altitude.ApproximatelyEquals(other.Altitude); 
         }
-
+        
+        public bool Equals(GeoPoint other, double epsilon)
```

Comment:
нужно сделать саммари
```suggestion
/// <summary>
/// Determines whether this geographic point is equal to another one within a specified tolerance.
/// </summary>
/// <param name="other">The other geographic point to compare with.</param>
/// <param name="epsilon">The allowed tolerance when comparing coordinates.</param>
/// <returns>
/// <c>true</c> if the longitude, latitude, and altitude differ by no more than the specified <paramref name="epsilon"/>; 
/// otherwise, <c>false</c>.
/// </returns>
public bool Equals(GeoPoint other, double epsilon)
```